### PR TITLE
#4081 - Better descriptions for row widths on flexilayouts

### PR DESCRIPTION
**Pull request by stpaultim**

### CODE REVIEW

**Comment by olafgrabienski** [comment]
Hm, I wanted to have a look at the sandbox site, but it isn't working.

**Comment by klonos** [comment]
@stpaultim please see my PR against yours: https://github.com/stpaultim/backdrop/pull/2

PS: FTR, the failures are those known false-positive, timezone-related ones; so unrelated to this PR 👍

File: core/modules/layout/layout.flexible.inc [hunk 1]
```diff
@@ -528,14 +523,23 @@ function layout_flexible_template_edit_row_form($form, &$form_state, LayoutFlexi
...
-      'container' => t("Fixed width (adds the 'container' row class)"),
-      'no_container' => t('Always full width of page (no row classes added)'),
+      'container' => t('Fixed maximum width'),
+      'container_fluid' => t('Fluid width'),
+      'no_container' => t('Full width'),
     ),
     '#default_value' => isset($row_data['container']) ? $row_data['container'] : 'container_fluid',
   );
+  $form['region_styles']['container']['container']['#description'] = t('Adds the <code>container</code> class to the row.');
+  $form['region_styles']['container']['container_fluid']['#description'] = t('Adds the <code>container-fluid</code> class to the row (no max-width, but padding).');
```
Comment (klonos) [inline core/modules/layout/layout.flexible.inc]: Wrap other code/CSS-related things in the description in code tags:

```suggestion
  $form['region_styles']['container']['container_fluid']['#description'] = t('Adds the <code>container-fluid</code> class to the row (no <code>max-width</code>, but <code>padding</code>).');
```

**Comment by quicksketch** [comment]
Thanks! Merged into 1.x and 1.14.x.

**Pull request merged**